lib/Auth/index.js

```diff
@@ -156,7 +156,7 @@ class Auth {
    *
    * @throws {CoreError} Returns erroror max authenticated request limit reached
    */
-  async send(service, method, url, opts, retry = 0) {
+  async send(service, method, url, opts, retry = 0, stream = false) {
```

Comment:
Let's add the new variable to `opts` and initialize the default value within Object.assign

```js
  /**
   * Send a request to Core service
....
   * @param {Object}  [opts.headers={}]     Headers to send
   * @param {Object}  [opts.query]          Query to send
   * @param {Object}  [opts.payload]        Payload to send
  *  @param {Object}  [opts.stream]
...
...
   */
  async send(service, method, url, opts, retry = 0) {
...
    opts = Object.assign(
      {
        ...
        ...
        stream: false
      },
     opts
    )
  }
```

Reply:
Moved stream to opts